contrib/packs/actions/create.meta.yaml

```diff
@@ -0,0 +1,41 @@
+---
+  name: "create"
+  runner_type: "action-chain"
+  description: "Creates a new pack"
+  enabled: true
+  entry_point: "workflows/create.yaml"
+  parameters:
+    name:
+      type: "string"
+      required: true
+      description: "Name of the pack to create"
+    description:
+      type: "string"
+      description: "Pack description"
+    keywords:
+      type: "array"
+      items:
+        type: "string"
+      description: "Search keywords"
+    version:
+      type: "string"
+      description: "Initial version of the pack"
+      required: true
+    author:
+      type: "string"
+      description: "Pack's author"
+      required: true
+    email:
+      type: "string"
+      description: "Pack's author email"
+    directories:
```

Comment:
I wonder if we should have an archived boilerplate instead, and then just unpack it and change `pack.yaml`. It would be really nice to place READMEs inside the dirs, like we used to do for old st2contrib packs, a default icon, etc. Does it seem reasonable?